src/TreeHouse/BaseApiBundle/Behat/ApiContext.php

```diff
@@ -321,12 +321,21 @@ public function theResultKeyShouldEqualNode($key, $node)
     }
 
     /**
-     * @Then the api error should contain ":text"
+     * @Then the api/API error should contain :text
```

Comment:
A case variant is not the same as providing synonyms for every word in the step definition.

Regarding the scope of this PR: the other steps also have this variant, so it's consistent with that.